pkg/controller/command/credentialclient/command_test.go

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package credentialclient
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/gorilla/mux"
+	"github.com/trustbloc/edge-agent/pkg/controller/command/sdscomm"
+	"github.com/trustbloc/edv/pkg/edvprovider/memedvprovider"
+	"github.com/trustbloc/edv/pkg/restapi"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestNew(t *testing.T) {
+	t.Run("Success", func(t *testing.T) {
+		c := New("", "")
+		require.NotNil(t, c)
+		require.NotNil(t, c.GetHandlers())
+	})
+}
+
+func TestCommand_SaveCredential(t *testing.T) {
+	t.Run("Success", func(t *testing.T) {
+		sdsSrv := newTestEDVServer(t)
+		defer sdsSrv.Close()
+
+		sampleCredentialData := sdscomm.CredentialData{}
+
+		credentialDataBytes, err := json.Marshal(sampleCredentialData)
+		require.NoError(t, err)
+
+		cmd := New(fmt.Sprintf("%s/encrypted-data-vaults", sdsSrv.URL), "James")
```

Comment:
I would pass "agentUsername" here. In my opinion, in tests it's always better when the string reveals the intention - so I don't need to refer to the function signature and check what "Jeff" is all about.

Reply:
Done.